libobs/obs-nix.c

```diff
@@ -66,8 +66,11 @@ static const int module_patterns_size =
 
 void add_default_module_paths(void)
 {
-	for (int i = 0; i < module_patterns_size; i++)
-		obs_add_module_path(module_bin[i], module_data[i]);
+	char on[] = "ON";
+	char *flag = getenv("OBS_IGNORE_BUILTIN_PLUGINS_PATH");
```

Comment:
It would be preferable to fix the problem rather than introducing a workaround no one will know about but you.

Reply:
As it happens only on obs shutdown, it's probably not very important. Also, it's not directly related to the purpose of this PR, hence I can just remove this commit from PR. WDYT?